api/editor/checkstyles.xml

```diff
@@ -3,6 +3,12 @@
           "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
           "https://checkstyle.org/dtds/configuration_1_3.dtd">
 
-<module name="SuppressionFilter">
-  <property name="file" value="${config_loc}/suppressions.xml" />
+<module name="Checker">
+  <module name="SuppressionFilter">
+    <property name="file" value="${config_loc}/suppressions.xml" />
+  </module>
+  <module name="HiddenField">
+    <property name="ignoreSetter" value="true" />
```

Comment:
Change the linting rules to allow functions to use the same names as class variables.